ofprotocol/ofp13/ofp13_parser.go

```diff
@@ -868,7 +868,7 @@ func (m *OfpPacketOut) Serialize() []byte {
 	actionLen := 0
 	aSize := 0
 	for _, a := range m.Actions {
-		actionLen += 1
+		actionLen += a.Size()
```

Comment:
Thank you @oqu.
ActionLen should be size of action array in BYTES as you say, and your proposal is effective.

However, In my original code, I set ActionLen value in other functions, which is NewOfpPacketOut and AppendAction function.
https://github.com/oqu/gofc/blob/2ce5abadaae59f3cd48dcbb35d98ba323174754a/ofprotocol/ofp13/ofp13_parser.go#L842
https://github.com/oqu/gofc/blob/2ce5abadaae59f3cd48dcbb35d98ba323174754a/ofprotocol/ofp13/ofp13_parser.go#L903

I think my code include redundant process, and may be remove some of these code in future, but if you can, could you fix that functions to set correct ActionLen value.